lib/rules.js

```diff
@@ -146,7 +147,49 @@ export const CANCEL_MINUSES = defineRuleString('-#a / -#b', '#a / #b')
 // CANCEL_TERMS: 'CANCEL_TERMS',
 
 // e.g. 2/6 -> 1/3
-// SIMPLIFY_FRACTION: 'SIMPLIFY_FRACTION',
+/*
+export const SIMPLIFY_FRACTION =
+    defineRuleString('#a / #b', '(#eval(#a/gcd(#a, #b)) / (#eval(#b/gcd(#a, #b))))', {a: query.isNumber, b: query.isNumber})
+*/
+export const SIMPLIFY_FRACTION = defineRule(
```

Comment:
I've update `math-evaluator`'s `gcd` to handle negatives and adding support for evaluating `abs` nodes.  The following works now:
```
export const SIMPLIFY_FRACTION = defineRuleString(
    '#a / #b',
    '#eval(sign(#a/#b) * |#a|/gcd(#a, #b)) / #eval(|#b|/gcd(#a, #b))',
    {a: query.isNumber, b: query.isNumber})
```
The only case it doesn't handle which is `0 / 3` which is already handled by `REDUCE_ZERO_NUMERATOR`.  I think shouldn't worry about this other case for now.  If we do need to handle as part of `SIMPLIFY_FRACTION`, https://github.com/semantic-math/math-rules/issues/10 should make it easy to combine rules.